Codeinwp/otter-blocks
# Add more fields type to Form block

Closes #1466.


### Summary


Added new fields:
- Select
- Checkbox
- Radio
- URL
- File (PRO)

Improvements:
- Better integration with other blocks in terms of spacing
- Detect fields in other blocks (Column, Section)

### Screenshots 

![image](https://user-images.githubusercontent.com/17597852/221148859-0628e86b-3802-4b45-b07d-d7900e832131.png)

#### Input Field detection & handling in other blocks

https://user-images.githubusercontent.com/17597852/221178927-2395e37a-1d72-46c6-a359-4a827545d7ba.mp4



----

### Test instructions


1. Add a Form and test new fields
2. Check the options in Sidebar
3. Make sure all the Fields works

ℹ️ For File Field accepted extension options, the user can write anything, but that does not mean the server will accept anything.

ℹ️ When it comes to input fields in other fields, the moving will happen at the direct parent and not individual (check the video, for example). Edit & Delete action will work at the individual level.

#### ⚠️ This will be an integration test. The new things added from the previous testing sessions are Autoresponder and Column Input width. 

#### ℹ️ The issue linked https://github.com/Codeinwp/otter-blocks/issues/1634 is more of a reminder; you can ignore it.



---- 

### Checklist before the final review
